packages/kbn-management/settings/components/form/services.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  FieldRowProvider,
+  FieldRowKibanaProvider,
+  type FieldRowKibanaDependencies,
+  type FieldRowServices,
+} from '@kbn/management-settings-components-field-row';
+import React, { FC } from 'react';
+
+export type FormServices = FieldRowServices;
```

Comment:
```suggestion
export interface FormServices {
  saveChanges: (changes: Record<string, UnsavedFieldChange<SettingType>>) => void;
  showError: (message: string) => void;
};
```

Reply:
Added this change with https://github.com/elastic/kibana/pull/166460/commits/ad18ae37177353621d2facc1ab96c766bb9dc8f6